janusgraph-cassandra/src/test/java/org/janusgraph/CassandraStorageSetup.java

```diff
@@ -124,7 +133,9 @@ public static WriteConfiguration getCassandraThriftGraphConfiguration(String ks)
      * from logging statements.
      */
     public static void startCleanEmbedded() {
-        startCleanEmbedded(getPaths());
+        if (getTestStorageHost() == null) {
```

Comment:
instead of testing getTestStorageHost here and elsewhere, can we just make it a required system property?